sdk/core/core/test/curl_easy_perform_poc.c

```diff
@@ -4,6 +4,8 @@
 #include <stdio.h>
 #include <curl/curl.h>
 
+#pragma warning(disable: 4996)
```

Comment:
please rebase against latest master, but in either case, you should've included _az_cfg.h instead of disabling.